internal/config/flags_test.go

```diff
@@ -26,14 +26,19 @@ func TestLoadFlags(t *testing.T) {
 		"9091",
 		"-metrics-port",
 		"9092",
+		"-origin-type",
+		"prometheus",
```

Comment:
do you want to test the origin url flag?

Reply:
origin test is already present higher up in the test